website/src/main.js

```diff
@@ -39,4 +39,25 @@ export default function(Vue, { router, head, isClient, appOptions }) {
     src: "/js/asciinema-player.js",
     body: true,
   });
+
+  head.link.push({
+    rel: 'stylesheet',
+    href: 'https://cdn.jsdelivr.net/npm/@algolia/algoliasearch-netlify-frontend@1/dist/algoliasearchNetlify.css'
+  })
+  head.script.push({
+    type: 'text/javascript',
+    body: true,
+    src: 'https://cdn.jsdelivr.net/npm/@algolia/algoliasearch-netlify-frontend@1/dist/algoliasearchNetlify.js'
+  })
+  head.script.push({
+    type: 'text/javascript',
+    body: true,
+    innerHTML: `algoliasearchNetlify({
+    appId: 'W8IEQZJ792',
+    apiKey: 'b16fa302b5c8c43816d11ecc7da702d2',
```

Comment:
Should this be secret?

Reply:
No, we don't need to make it secret because it is only against the specific app it can't be used in other apps Thanks!

Reply:
Not sure how they would know how to distinguish one app from another if all they have is the API key.

Reply:
Okay this apId and secret are generated against a specific app like in our case 'Talos' from the Algolia Search dashboard so once these keys are added then Algolia dashboard detects the App but if you say I can add this in env so where you place the key share me the process.env.key_name I will add Thanks

Reply:
Right. But the API key _is_ the identifier I imagine. Putting it in the env is a good idea. We will need to generate the key again since it was exposed.

Reply:
Sure can you share the file name where i need to place appId & apiKey so after that i will access in code with its key name Thanks

Reply:
E.g process.env.GA_ID or process.env.SITE_URL

Reply:
I can set an env var in netlify. If you can code it to use process.env I can the rest up.

Reply:
Okay here are the keys with their values you need to set in .env 

1- GRIDSOME_ALGOLIA_APP_ID  = W8IEQZJ792

2- GRIDSOME_ALGOLIA_API_KEY  = b16fa302b5c8c43816d11ecc7da702d2

3- GRIDSOME_ALGOLIA_SITE_ID  =  4ce8d530-06a7-45d7-bc2a-0484644882df

so I will access like as process.env.GRIDSOME_ALGOLIA_APP_ID e.t.c

Let me know then Thanks!

Reply:
I have commented the code with secret keys for Algolia Search kindly review Thanks!